routes/v1/users.js

```diff
@@ -86,21 +86,21 @@ module.exports = function(/*middleware*/) {
 				return false;
 			}
 
-			var timestamp = parseInt(req.body.timestamp, 10) || undefined;
+			var timestamp = parseInt(req.body.timestamp, 10) || Date.now();
```

Comment:
If I don't set the timestamp explicitly, the message will not show in the chat history.